### PR TITLE
Update DG (Delete)

### PR DESCRIPTION
**Pull request by bhnuka**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -661,16 +732,33 @@ testers are expected to do more *exploratory* testing.
 
    1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
 
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
```
Comment (syamfarh) [inline docs/DeveloperGuide.md:806]: just a question, does the timestamp refer to the red messages in the cmd, or the one shown in the result box?

Comment (bhnuka) [reply docs/DeveloperGuide.md:806]: Honestly i wasnt too sure so i just deleted it haha

Comment (syamfarh) [reply docs/DeveloperGuide.md:806]: alright I think I delete from sort as well

**Comment by syamfarh** [comment]
LGTM!

**Pull request merged**